src/EDMF_functions.jl

```diff
@@ -270,7 +270,14 @@ end
 function w_surface_bc(::SurfaceBase{FT})::FT where {FT}
     return FT(0)
 end
-function θ_surface_bc(surf::SurfaceBase{FT}, grid::Grid, state::State, edmf::EDMFModel, i::Int)::FT where {FT}
+function θ_surface_bc(
+    surf::SurfaceBase{FT},
+    grid::Grid,
+    state::State,
+    edmf::EDMFModel,
+    i::Int,
+    n_updrafts::Int,
+)::FT where {FT}
```

Comment:
```suggestion
function θ_surface_bc(surf::SurfaceBase{FT}, grid::Grid, state::State, edmf::EDMFModel, i::Int)::FT where {FT}
    n_up = n_updrafts(edmf)
```
We're already passing `edmf`, so we can grab `n_up` from `n_updrafts(::EDMFModel)` I think this will help make the diff much smaller, and add fewer lines.